internal/internal/db/record.go

```diff
@@ -0,0 +1,37 @@
+package locodedb
+
+// Record represents a single record of the UN/LOCODE table.
+type Record struct {
+	// Change Indicator.
+	Ch string
```

Comment:
what is it for? how does it look? not used?

Reply:
removed